eppoclient/application_logger.go

```diff
@@ -0,0 +1,11 @@
+package eppoclient
+
+import "regexp"
+
+// maskSensitiveInfo replaces sensitive information (like apiKey or sdkKey)
+// in the error message with 'XXXXXX' to prevent exposure of these keys in
+// logs or error messages.
+func maskSensitiveInfo(errMsg string) string {
+	re := regexp.MustCompile(`(apiKey|sdkKey)=[^&]*`)
```

Comment:
super minor but conisder `+` instead of star. That way, if the key is empty, we'd log that it was empty, which in some scenarios _could_ be useful.